index.html

```diff
@@ -57,11 +57,227 @@ <h3 class="connect">LET'S CONNECT</h3>
+            <img src="./images/Snapshoot-Portfolio3.png" alt="" />
+
+            <div class="card-info">
+              <h2>Multi-Post Stories</h2>
+
+              <div class="title-and-date">
+                <p class="title">CANOPY</p>
+                <span></span>
+                <p class="muted">BACK END DEV</p>
+                <span></span>
+                <p class="muted">2015</p>
+              </div>
+
+              <p class="description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+
+              <ul class="stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+
+              <a class="btn" href="#">See projects</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- MYSELF -->
+    <section class="about-bg">
+      <div class="myself about container">
+        <h1>About Myself</h1>
+        <p>
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+        <h3>LET'S CONNECT</h3>
+
+        <ul>
+          <li>
+            <a href="https://twitter.com/Peekay_Codes"
+              ><i class="uil uil-twitter"></i
+            ></a>
+          </li>
+
+          <li>
+            <a href="https://www.linkedin.com/in/prosper-kessie-363968171/"
+              ><i class="uil uil-linkedin"></i
+            ></a>
+          </li>
+
+          <li>
+            <a href="#"><i class="uil uil-medium-m"></i></a>
+          </li>
+
+          <li>
+            <a href="https://github.com/kessie2862"
+              ><i class="uil uil-github"></i
+            ></a>
+          </li>
+
+          <li>
+            <a href="#"><img src="./images/Vector.svg" alt="Two-fingers" /></a>
+          </li>
+        </ul>
+
+        <a class="btn btn-self" href="#">Get my resume</a>
+
+        <!-- SKILLS -->
+        <div class="skills">
+          <div class="categories">
+            <div class="dropdown">
+              <h3>Languages</h3>
+              <i class="uil uil-angle-down"></i>
+            </div>
+
+            <div class="dropdown-items">
+              <ul>
+                <img src="./images/Ellipse1.png" alt="" />
+                <li>Javascript</li>
+              </ul>
+            </div>
+
+            <div class="dropdown-items">
+              <ul>
+                <img src="./images/Ellipse2.png" alt="" />
+                <li>HTML</li>
+              </ul>
+            </div>
+
+            <div class="dropdown-items last">
+              <ul>
+                <img src="./images/Ellipse3.png" alt="" />
+                <li>CSS</li>
+              </ul>
+            </div>
+          </div>
```

Comment:
- Again, you have done well so far, but to avoid using so many `div` tags in your project and ensure that you use more semantic tags, I suggest that you wrap this section in a `ul` tag and wrap each corresponding technology in a `li` tag. Please fix this 😄

Reply:
Resolved